app/components/topic-card/component.js

```diff
@@ -14,8 +15,8 @@ export default Ember.Component.extend({
   isUpvoter: Ember.computed('session.currentUser', 'topic.upvoters.[]', function isUpvoter() {
     return this.get('topic.upvoters').contains(this.get('session.currentUser'));
   }),
-  isVolunteer: Ember.computed('session.currentUser', 'topic.volunteers.[]', function isVolunteer() {
-    return this.get('topic.volunteers').contains(this.get('session.currentUser'));
+  talkByCurrentUser: Ember.computed('session.currentUser', 'topic.talkBy', function() {
+    return this.get('session.currentUser.username') === this.get('topic.talkBy.username');
```

Comment:
Not that I know of, were you using `Ember.computed.equal('session.currentUser', 'topic.talkBy')` or `Ember.computed.equal('session.currentUser.username', 'topic.talkBy.username')`?

Also, if we go the route that you have here instead of using `Ember.computed.equal`, then I think the `Ember.computed` properties should match the `this.get` properties.